### PR TITLE
document warpx example

### PR DESCRIPTION
**Pull request by RTSandberg**
This PR updates and lists the WarpX-optimas example in the online documentation.

### CODE REVIEW

File: doc/source/examples/bo_with_warpx.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+=======================
+
+
+Description
+~~~~~~~~~~~
+
+This examples shows how to perform a Bayesian optimization of a laser-plasma
+accelerator (LPA) using WarpX simulations.
+
+The LPA to be optimized is based on
```
Comment (RemiLehe) [inline doc/source/examples/bo_with_warpx.rst:13]: This sentence seems incomplete.

File: doc/source/examples/bo_with_warpx.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+
+where :math:`\epsilon_i` and :math:`\epsilon_f` are the initial and final beam emittances, respectively,
+and :math:`Q_i` and :math:`Q_f` are the initial and final beam charges.
+This objective is optimized by tuning 2 parameters:
+
+- ``'adjust_factor'``: parameter in the range :math:`[0.7, 1.05]` that scales the
+  strength of the magnetic field between the first and second stage.
+  The value ``adjust_factor=1`` corresponds to a focusing strength of :math:`454535.7\, \mathrm{T/m}`.
+- ``'zlen'``: the left or starting position position of the laser in millimetres, with range
```
Comment (RemiLehe) [inline doc/source/examples/bo_with_warpx.rst:28]:
```suggestion
- ``'zlen'``: the position of the focusing plasma lens in millimetres, with range
```

**Review by RemiLehe — CHANGES_REQUESTED** [review]
Thanks for this PR! I added a few minor comments.

File: doc/source/examples/bo_with_warpx.rst (hunk 1 above)
Comment (RTSandberg) [reply doc/source/examples/bo_with_warpx.rst:13]: Yep, that was a stub I meant to delete.  It's gone now

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**